README.md

```diff
@@ -9,8 +9,11 @@ Using your AWS API credentials, S3Proxy allows you to make unauthenticated HTTP
 
 This makes it easy for tools like curl to access private buckets:
 
+Eg. Assuming you have a S3 bucket named `bucket1` and it has the following 
+structure `folder1/folder2/item.txt`. 
+
 ```bash
-curl http://localhost:9090/object/key
+curl http://localhost:9090/bucket1/folder1/folder2/item.txt?aws_region=eu-west-1
```

Comment:
Also, then there can be a default region and it can be overridden by CLI parameters or config files.